### PR TITLE
Allow AMInstallButton to work with a null currentVersion

### LINKED ISSUE #6976: The product detail page for a non public addon is not displayed for admins and developers
Followup for https://github.com/mozilla/addons-frontend/issues/6944 and https://github.com/mozilla/addons-frontend/issues/4244

STR:
1. Load AMO dev and submit an add-on, wait for it's approval.
2. As a developer disable it's first version
3. Click on View Listing link

Expected result:
The detail page is shown for the developer with the message "This is not a public listing. You are only seeing it because of elevated permissions."

Actual result:
The page is not displayed. 

Notes:
This is not reproducible once the version is re-enabled  - but it's because of https://github.com/mozilla/addons-server/issues/3051.
Also when the add-on is disabled by an admin from rev tools the issue is not reproducible.
Reproducible on AMO dev and AMO stage with FF63(Win10).

disabled - first version

### PR DESCRIPTION
**Pull request by bobsilverberg**
Fixes #6976

### CODE REVIEW

File: src/core/components/AMInstallButton/index.js [hunk 1]
```diff
@@ -308,7 +310,7 @@ export class AMInstallButtonBase extends React.Component<InternalProps> {
     const buttonProps: ButtonProps = {
       buttonType: 'action',
       className: 'AMInstallButton-button',
-      disabled: buttonIsDisabled,
+      disabled: buttonIsDisabled || !installURL,
```
Comment (willdurand) [inline src/core/components/AMInstallButton/index.js]: Why not adding this condition to the `buttonIsDisabled` above?

Comment (bobsilverberg) [reply src/core/components/AMInstallButton/index.js]: When I first tried to do that, it seemed to be getting unwieldy with the ternary, but I took another look and reversed the logic and now it seems okay to just add it into `buttonIsDisabled`, so I've done that.

**Comment by bobsilverberg** [comment]
Thanks @willdurand!

**Comment by willdurand** [comment]
Thanks for the quick fix!

**Pull request merged**